openqa-trigger-bisect-jobs

```diff
@@ -299,10 +299,22 @@ def main(args):
             ]
         )
 
-        out = openqa_clone(
-            params,
-            args.dry_run,
-        )
+        try:
+            out = openqa_clone(
+                params,
+                args.dry_run,
+            )
+        except subprocess.SubprocessError as err:
+            process_output = err.output if err.output else ""
+            log.error(f"{type(err).__name__}: {process_output} ")
+            if 'the repositories for the below updates are unavailable' not in str(process_output):
+                raise
+            else:
+                sys.exit(0)
+        except Exception as err:
+            err.add_note("openqa_clone failed on unknown cause.")
+            raise
```

Comment:
why is this necessary. Isn't any not catched exception raised further anyway?

Reply:
I think @perlpunk comment was only about the `if` case in the `SubprocessError` handler. We don't need to add a generic handler just to log an additional error message.